templates/ZF/mnc_5stage/train.prototxt

```diff
@@ -763,7 +763,7 @@ layer {
 		lr_mult: 2.0 
 	}
 	inner_product_param {
-		num_output: 8
+		num_output: {{ (num_classes|int) * 4 }}
```

Comment:
Is the forced conversion to int really a good idea? The conversion can't fail and will return 0 if the input was garbage. If you don't convert it to int then invalid values may at least cause the prototext parsing to fail instead of silently using 0.